google/cloud/security/common/gcp_api/cloud_resource_manager.py

```diff
@@ -114,16 +114,17 @@ def get_project_iam_policies(self, resource_name, project_numbers):
         """
         projects_stub = self.service.projects()
 
-        try:
-            for project_number in project_numbers:
+        for project_number in project_numbers:
```

Comment:
Very good points.  I have addressed the core of this comment by moving the loop to the caller, and retain the APIExecutionError.

The only item that's tricky to do, is to genericize the incoming project_number parameter.  Because the project_number is used specifically as the key in the project_iam_policies database table.
https://github.com/GoogleCloudPlatform/forseti-security/blob/master/google/cloud/security/common/data_access/sql_queries/create_tables.py#L38
How do you think?